Extensions/Widgets/ViewModels/MenuWidgetViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OrchardCore.Navigation;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Lombiq.HelpfulExtensions.Extensions.Widgets.ViewModels
+{
+    public class MenuWidgetViewModel
+    {
+        public bool NoWrapper { get; set; }
+        public IEnumerable<MenuItem> MenuItems { get; set; }
+
+        public MenuWidgetViewModel(bool noWrapper = false, IEnumerable<MenuItem> menuItems = null)
+        {
+            NoWrapper = noWrapper;
+            MenuItems = menuItems ?? Enumerable.Empty<MenuItem>();
+        }
+
+        public MenuWidgetViewModel(dynamic model)
+            : this(model.NoWrapper as bool? ?? false, model.MenuItems as IEnumerable<MenuItem>)

```

Comment:
@DAud-IcI What do you think of:
```suggestion
            : this((bool)(model.NoWrapper == true), model.MenuItems as IEnumerable<MenuItem>)
```

Reply:
It confuses the IDE (it thinks the `== true` is not necessary even though it is) so I'd rather not, to avoid future problems.

Reply:
Now I remember - this is what I had in mind:

```suggestion
            : this(model.NoWrapper as bool? == true, model.MenuItems as IEnumerable<MenuItem>)
```

I find this more intuitive than the `?? false`. What do you think?

Reply:
Sure, that works. Though you also need to add parenthesis.

Reply:
I didn't have to in LINQPad:

![image](https://user-images.githubusercontent.com/164360/150878896-9587d4df-3765-4f17-bd4d-7c15351713d2.png)

😕

Reply:
Yeah it's a code analysis warning. You don't have those in LINQPad.